### ISSUE TITLE
No target found in `target-string` causes IndexError

### ISSUE THREAD

**Issue by tomjw64** [openstax/cnx-easybake#99]
```html
<html>
  <a></a>
</html>
```
baked with
```css
a::before {
  content: target-string(attr(some-attr), some-counter)
}
```
yields
```
Traceback (most recent call last):
  File "../cnx-recipes/venv/bin/cnx-easybake", line 11, in <module>
    load_entry_point('cnx-easybake', 'console_scripts', 'cnx-easybake')()
  File "/home/thomas/code/openstax/sandbox/cnx-recipes/venv/src/cnxeasybake/cnxeasybake/scripts/main.py", line 88, in main
    args.coverage_file, args.use_repeatable_ids)
  File "/home/thomas/code/openstax/sandbox/cnx-recipes/venv/src/cnxeasybake/cnxeasybake/scripts/main.py", line 20, in easybake
    oven.bake(html_doc, last_step)
  File "/home/thomas/code/openstax/sandbox/cnx-recipes/venv/src/cnxeasybake/cnxeasybake/oven.py", line 275, in bake
    recipe = self.build_recipe(wrapped_html_tree, step)
  File "/home/thomas/code/openstax/sandbox/cnx-recipes/venv/src/cnxeasybake/cnxeasybake/oven.py", line 423, in build_recipe
    _state = self.build_recipe(el, step, depth=depth+1)  # noqa
  File "/home/thomas/code/openstax/sandbox/cnx-recipes/venv/src/cnxeasybake/cnxeasybake/oven.py", line 417, in build_recipe
    method(element, decl, 'before')
  File "/home/thomas/code/openstax/sandbox/cnx-recipes/venv/src/cnxeasybake/cnxeasybake/oven.py", line 41, in with_logging
    return func(*args, **kwargs)
  File "/home/thomas/code/openstax/sandbox/cnx-recipes/venv/src/cnxeasybake/cnxeasybake/oven.py", line 1118, in do_content
    target_args[0])[0]
IndexError: list index out of range
```

Ideally, it should fail a bit more gracefully, like
```css
a::before {
  content: target-string(attr(some-attr, "#"), some-counter)
}
```
which yields
```
cnx-easybake WARNING Bad ID target lookup at 2,3 : ""
```

**Comment by brittweinstein** [comment]
@tomjw64 what milestone should this go in?

**Comment by tomjw64** [comment]
@brittweinstein No idea. This is non-blocking and low priority if that helps. I just logged it when I saw it a while ago. It is easy to work around in recipes.

**Comment by tomjw64** [comment]
Resolved by #109
